test/unit/presenters/publishing_api/html_attachment_presenter_test.rb

```diff
@@ -21,7 +21,7 @@ def present(record)
     html_attachment = HtmlAttachment.last
 
     expected_hash = {
-      base_path: "/government/publications/#{edition.document.slug}/#{html_attachment.slug}",
+      base_path: "/government/publications/#{edition.document.slug}/#{html_attachment.content_id}",
```

Comment:
The English HTML attachments should retain the title-to-slug behaviour. We only want to apply the `content_id` to non-English HTML attachments (in place of their edition ID).